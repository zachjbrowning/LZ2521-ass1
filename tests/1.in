s z
s a
s ajouter
s re
s utiliser
q
