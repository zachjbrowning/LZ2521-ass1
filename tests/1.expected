> > ajouter to add
aller to go
appeler to call
arriver to arrive
attendre to wait
avoir to have
> ajouter to add
> recevoir to receive
rendre to return
> > 